sdk/communication/communication-job-router/README.md

```diff
@@ -0,0 +1,396 @@
+| Distribution Policy | The distribution policy is a container of rules on how jobs are allocated and distributed to workers from a queue. This includes offer time to live, distribution method (e.g. round robin, top (n) available agents, etc) and allocation rules. |
+| Exception Policy | A container of rules that define what action to take when a particular exception occurs.  This container of rules can be associated with one or more queues. |
+
+##Setting Up
+###Install an IDE such as [VSCode](https://code.visualstudio.com/download) or [Webstorm](https://www.jetbrains.com/webstorm/download/) if you haven't (Optional)
+
+###Install NodeJS if you haven't
+LTS or Current version, either is ok.
+https://nodejs.org/en/download/
+
+###Create a new NodeJS Express server
+In a console window (such as cmd, PowerShell, or Bash), create a new folder named RouterQuickStart and use `npx express-generator` to create a new Express app in the current folder. This will create a simple "Hello World" Express project that will listen on port 3000
+```
+npx express-generator
+npm install
+DEBUG=routerquickstart:* npm start
+```
+
+###Create an ACS resource under dogfood
+Refer to this doc, https://skype.visualstudio.com/SPOOL/_wiki/wikis/SPOOL.wiki/20243/Azure-dogfood-env
+Create a new resource or use existing resource. If creating a resource, select acs-pstn-dogfood as resource group.
+
+![image.png](/.attachments/image-1a0118bb-0063-420b-924a-943b66ef0d01.png)
+
+###Install the Azure Communication Services Router SDK
+Create a file .npmrc to your project. Put following code into it:
+```
+@azure-ccaap:registry=https://pkgs.dev.azure.com/azure-sdk/internal/_packaging/azure-sdk-for-js-pr/npm/registry/ 
+registry=https://registry.npmjs.org
+                        
+always-auth=true
+```
+
+Then, run vsts-npm-auth to get an Azure Artifacts token added to your user-level .npmrc file
+
+`vsts-npm-auth -config .npmrc`
+
+If it says vsts-npm-auth not found, then you need to install it first:
+
+`npm install -g vsts-npm-auth`
+
+
+While still in the application directory, install the Azure Communication Services Router client library for Javascript package by using the npm install --save command
+`npm install --save @azure-ccaap/communication-router`
+
+## Examples
+###Initialize Router Client
+First we need to initialize a Router client.
+```js
+import { RouterClient } from '@azure/communication-router';
+
+const acsConnectionString = 'endpoint=https://<YOUR_ACS>.communication.azure.com/;accesskey=<YOUR_ACCESS_KEY>'
+const routerClient = new RouterClient(acsConnectionString );
+```
+
+##Configure Queue and Workers
+###Create a queue
+Then we create a sales queue with various labels.
+```js
+const salesQueueResponse = await routerClient.CreateQueue({
```

Comment:
Generally we try to make snippets in the READMEs copy-pasteables, meaning with the imports and a necessary async function in order for people to just copy and paste it in a file, without having to look elsewhere.

I am assuming this can be done in the future! So I won’t be suggesting doing that change in this PR.